src/pacasam/connectors/lipac.py

```diff
@@ -90,14 +91,16 @@ def download_database(self, extraction_sql_query: str, max_chunksize_for_postgis
         gdf = gdf.set_crs(self.lambert_93_crs)
```

Comment:
Not related to this pr but: why forcing to lambert_93? (what if we deal with data with other crs?)

Reply:
In its current state, LiPaC only contains L93 data, and was conceived with a fixed number of lidar blocks in mind.
This being said, good catch: I think we integrated the SRID in LiPaC and if so we should use it! I'll check and make a separate PR if this ius the case :)

Reply:
Oups. I made a fix, simply deleting the involuntarily pushed on main directly. I simply do not set the crs, since it is not used during sampling, and actually provided via a `srid` value of the sampling geopackage for extraction.
Mais consequence should be that one has to set the crs when opening the fil in e.g. QGIS.
I bumped the patch version in consequence.